src/libraries/icubmod/embObjMultipleFTsensor/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2022 Istituto Italiano di Tecnologia (IIT)
+# All rights reserved.
+# Author: Luca Tricerri
+# This software may be modified and distributed under the terms of the
+# BSD-3-Clause license. See the accompanying LICENSE file for details.
+
+yarp_prepare_plugin(embObjMultipleFTsensor CATEGORY device
+                                 TYPE yarp::dev::embObjMultipleFTsensor
+                                 INCLUDE embObjMultipleFTsensor.h
+                                 EXTRA_CONFIG WRAPPER=analogServer)
+
+IF (NOT SKIP_embObjMultipleFTsensor)
+
+
+  set(ICUB_COMPILE_EMBOBJ_LIBRARY ON CACHE INTERNAL "use the embObjLib lib")
```

Comment:
I guess this was a result of copy&apste, but it does not make a lot of sense to have this option here.